content/en/v3/admin/guides/k8s-upgrades/_index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Kubernetes upgrades
+linktitle: Kubernetes upgrade
+type: docs
+description: How to upgrade the version of Kubernetes in your cluster.
+weight: 92
+---
+ 
+Upgrading your Kubernetes cluster is one of the tasks you must take on as an operator of Jenkins X. Kubernetes comes out with [new releases](https://kubernetes.io/releases/) regularly, and will only maintain releases for the three latest versions. Meaning if something breaks, or there is a security issue on an older version, Kubernetes will not fix it, but ask you to upgrade.  All the major cloud providers only support a given set of latest versions. 
```

Comment:
Should we mention that it is good to test an upgrade before one upgrades in prod?  Also, what about downgrade? Do we have DR for jenkins-x?  What to do if an upgrade fails?

Reply:
Downgrade, I dont think that is supported, really. The solution then is to delete all and start from scratch. Of course that will be quick because gitops.